pkg/util/redirect.go

```diff
@@ -0,0 +1,36 @@
+package util
+
+import (
+	"os"
+	"sync"
+)
+
+var (
+	redirectMap = &RedirectStroe{rs: sync.Map{}}
```

Comment:
Sorry but passing around pipes by stuffing them in a global variable is not acceptable.

Reply:
Only in windows system need to use the pipeline...
Using `debugger.Config` to pass the pipeline will change the definition of the `native.Launch` function. 
Is this acceptable to you?

Reply:
Yes. Also this will eventually need tests, both the nodebug path and the normal path. And Debugger.Restart needs to return an error in case pipe redirects are used (it would be even better if it worked correctly instead of erroring but that's hard and dap doesn't call Restart anyway).